spec/requests/citizens/additional_accounts_spec.rb

```diff
@@ -16,6 +16,19 @@
     end
   end
 
+  context 'when an applicant revisits the page to change their answer' do
+    before do
+      application.update!(has_offline_accounts: true)
+      application.update!(state: 'use_ccms')
+    end
```

Comment:
it's OK to update the state directly in tests if there is no other way.  this could be combined into one statement, e.g. 
`application.update!(has_offline_accounts: true, state: 'use_ccms')`